scalableminds/webknossos
# Request Error Logging

I added console logging for non-200 answers. Newrelic notices these but does not provide the actual error message. I added it to all the tracingstore routes as that is where most confusing errors occured recently. Where else do you think it would make sense? What other information do you think would make sense to include in the log? Should we also attempt to send it to newrelic?

### Steps to test:
- change some requests that have the `log` block so that they will reply with bad request or similar (e.g. insert a Fox.failure)
- request should be listed on the command line along with the error message that is returned to the client

### Issues:
- fixes #3510 

------
